src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/CreateCohortController.cs

```diff
@@ -30,15 +34,21 @@ public class CreateCohortController : Controller
             IMapper<IndexRequest, IndexViewModel> indexViewModelMapper,
             IMapper<SelectProviderRequest, SelectProviderViewModel> selectProviderViewModelMapper,
             IMapper<SelectProviderViewModel, ConfirmProviderRequest> confirmProviderRequestMapper,
+            IMapper<ConfirmProviderRequest, ConfirmProviderViewModel> confirmProviderViewModelMapper,
+            IMapper<ConfirmProviderViewModel, SelectProviderViewModel> selectProviderFromConfirmMapper,
             IValidator<SelectProviderViewModel> selectProviderViewModelValidator,
+            IValidator<ConfirmProviderViewModel> confirmProviderViewModelValidator,
```

Comment:
We should be able to remove the Validators

Reply:
done